tests/end-to-end/ui/11-admin.js

```diff
@@ -731,8 +731,8 @@ describe('[Administration]', () => {
 					admin.accountsEnableAutoAwayFalse.isVisible().should.be.true;
 				});
 				it('the enable auto away field value should be true', () => {
```

Comment:
it makes sense now.. the test description is correct, but the test was wrong. 🤦‍♂️